index.bs

```diff
@@ -1957,6 +1977,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain fake probability for a source with attribution scope</dfn> given a positve 32-bit integer |numSourceStates|, a positve 32-bit integer |numEventStates|, a positve 32-bit integer |numEvents|, a positive and a double |epsilon|:
+1. Let |fakeProbability1| be (|numSourceStates| - 1) / (|numSourceStates| - 1 + e<sup>|epsilon|</sup>).
```

Comment:
Can we use more descriptive terms than `fakeProbability`? Like pick rate for click and pick rate for view?